src/Microsoft.ML.Data/Scorers/GenericScorer.cs

```diff
@@ -114,7 +114,7 @@ public Bindings ApplyToSchema(IHostEnvironment env, Schema input)
                 return Create(env, bindable, input, roles, suffix, user: false);
             }
 
-            public override void Save(ModelSaveContext ctx)
+            internal override void SaveModel(ModelSaveContext ctx)
```

Comment:
>internal [](start = 12, length = 8)

`private protected` if possible.

Reply:
In this case it is not possible, because `BindingsBase` is not `ICanSaveModel`, and other classes need to access this method. However, I went through all the instance of `internal.* Save(ModelSaveContext...` and fixed a couple places where we could make private protected, so thank you for pointing that out.

---
In reply to: [254963789](https://github.com/dotnet/machinelearning/pull/2431#discussion_r254963789) [](ancestors = 254963789)